activerecord/lib/active_record/enum.rb

```diff
@@ -117,10 +117,11 @@ def inherited(base) # :nodoc:
     class EnumType < Type::Value # :nodoc:
       delegate :type, to: :subtype
 
-      def initialize(name, mapping, subtype)
+      def initialize(name, mapping, subtype, validate: true)
```

Comment:
Having this enabled by default would be a breaking change?

Reply:
Yes, changing the default will break the existing test here:

https://github.com/rails/rails/blob/97832cb4b2dfce82dcb096ea83985555b4d14c5b/activerecord/test/cases/enum_test.rb#L295-L300

Reply:
@MatheusRich i found it a bit confusing too, but `validate` here is the opposite to the `validate` prop you'd provide in your model. So validate will be `true` here if the validate prop is `false` (or not present) on the model's call to `enum`.

See https://github.com/rails/rails/pull/41730/files#diff-a0de22dc0114613fdc67f48c916e70b9983ef47bcc84b96232c36398af1d166fR190

![image](https://user-images.githubusercontent.com/509837/112884918-b5d39000-9095-11eb-8e6d-0e9c12eec997.png)

Reply:
```suggestion
      def initialize(name, mapping, subtype, raise_on_invalid_values: true)
```